packages/backend/src/oauth.ts

```diff
@@ -0,0 +1,90 @@
+const OAuth = (oauthCallback: string) => {
+    const CONSUMER_KEY = process.env.CONSUMER_KEY
+    const CONSUMER_SECRET = process.env.CONSUMER_SECRET
+    const _oauth = new (require('oauth').OAuth)(
```

Comment:
Is this oAuth implementation using oAuth1 or oAuth2? I looked at the docs but not entirely sure this is oAuth2.